webapp/src/reducers.ts

```diff
@@ -26,30 +26,29 @@ import {
     SHOW_EXPANDED_VIEW,
     SHOW_SCREEN_SOURCE_MODAL,
     SHOW_SWITCH_CALL_MODAL,
-    VOICE_CHANNEL_CALL_END,
-    VOICE_CHANNEL_CALL_HOST,
-    VOICE_CHANNEL_CALL_REC_PROMPT_DISMISSED,
-    VOICE_CHANNEL_CALL_RECORDING_STATE,
-    VOICE_CHANNEL_CALL_START,
-    VOICE_CHANNEL_PROFILE_CONNECTED,
-    VOICE_CHANNEL_PROFILES_CONNECTED,
-    VOICE_CHANNEL_ROOT_POST,
-    VOICE_CHANNEL_UNINIT,
-    VOICE_CHANNEL_USER_CONNECTED,
-    VOICE_CHANNEL_USER_DISCONNECTED,
-    VOICE_CHANNEL_USER_MUTED,
-    VOICE_CHANNEL_USER_RAISE_HAND,
-    VOICE_CHANNEL_USER_REACTED,
-    VOICE_CHANNEL_USER_REACTED_TIMEOUT,
-    VOICE_CHANNEL_USER_SCREEN_OFF,
-    VOICE_CHANNEL_USER_SCREEN_ON,
-    VOICE_CHANNEL_USER_UNMUTED,
-    VOICE_CHANNEL_USER_UNRAISE_HAND,
-    VOICE_CHANNEL_USER_VOICE_OFF,
-    VOICE_CHANNEL_USER_VOICE_ON,
-    VOICE_CHANNEL_USERS_CONNECTED,
-    VOICE_CHANNEL_USERS_CONNECTED_STATES,
-    VOICE_CHANNEL_USER_JOINED_TIMEOUT,
+    CALL_END,
+    CALL_REC_PROMPT_DISMISSED,
```

Comment:
I stripped most of the unnecessary prefixes in this file with the assumption that it's all Calls related. Please let me know if you'd prefer a more verbose take on this side as well.

Reply:
Definitely this is better, thanks!